src/core/client/admin/test/components/UserHistoryDrawer/UserHistoryDrawer.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  createSettings,
+  createStory,
+  createUser,
+} from "coral-admin/../test/helpers/fixture";
```

Comment:
Fix weird pathing.

Reply:
Would you be okay if I added a new named path called `coral-client` to remedy this?

Reply:
Nevermind, it looks like there is a `coral-test` to be used.